spack/repo/packages/py-pyprecice/package.py

```diff
@@ -15,11 +15,13 @@ class PyPyprecice(PythonPackage):
     homepage = "https://www.precice.org"
     git = "https://github.com/precice/python-bindings.git"
     url = "https://github.com/precice/python-bindings/archive/v2.0.0.1.tar.gz"
-    maintainers = ["ajaust", "BenjaminRodenberg"]
+    maintainers = ["ajaust", "BenjaminRodenberg", "IshaanDesai"]
```

Comment:
Already confirmed in https://github.com/precice/python-bindings/pull/115#discussion_r685818194.